src/components/metadata/CollectionMetadata.tsx

```diff
@@ -21,8 +21,8 @@ const CollectionMetadata: FC = () => {
 
   return <Accordion type="single" collapsible>
     <AccordionItem value="item-1" >
-      <AccordionTrigger className="t-py-1 t-relative [&>svg]:t-absolute [&>svg]:t-right-3">{ t('collection') }</AccordionTrigger>
-      <AccordionContent><div className="t-ml-2">{ metadata.map((meta, i) => <MetadataItem item={meta} key={i} />)}</div>
+      <AccordionTrigger className="t-py-[4px] t-pl-3 t-pr-1 t-mr-[6px] t-rounded-md t-bg-gray-100 hover:t-no-underline hover:t-bg-gray-200 t-mb-1 t-text-gray-700 t-font-bold">{ t('collection') }</AccordionTrigger>
```

Comment:
Please follow [DRY](https://de.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself) and implement it directly in the accordion comps